validation/src/test/scala/hmda/validation/rules/ts/validity/ValidityUtils.scala

```diff
@@ -0,0 +1,24 @@
+package hmda.validation.rules.ts.validity
+
+import hmda.model.fi.ts.Respondent
+
+trait ValidityUtils {
+  def respondentNotEmpty(r: Respondent): Boolean = {
```

Comment:
I still believe this to be true, but I intend to merge anyway. Perhaps if a later user of the code is confused by the inconsistency, they will have good ideas for what to do about it.